util/random/rand.go

```diff
@@ -76,3 +81,59 @@ func (r *randstream) XORKeyStream(dst, src []byte) {
 func New() cipher.Stream {
 	return &randstream{}
 }
+
+// READER_BYTES is how many bytes we expect from each source
+const READER_BYTES = 32
+
+// MIXEDRAND_HASH is the hash used in XORKeyStream to mix all entropy sources together.
+// sha512 is chosen as its digest is 64 bytes long, matching the length of blake2xb's seed.
+var MIXEDRAND_HASH = sha512.New
```

Comment:
I'm guessing one reason Travis fails is because in Go we don't use UPPER case for constants, we use regular CamelCase.